explorer/src/utils/explorer.ts

```diff
@@ -135,6 +138,9 @@ const nativeExplorerContractUri = (
       base = "https://ftmscan.com/address/";
     } else if (chainId === chainIDs["aurora"]) {
       base = "https://aurorascan.dev/address/";
+    } else if (chainId === chainIDs["alephium"]) {
+      // FIXME
+      base = "https://explorer.testnet.alephium.org/address"
```

Comment:
Is this fixed?

Reply:
Yes, it has been fixed here: https://github.com/alephium/wormhole-fork/pull/59/files#diff-8b4475005d0f7c6a2c534389d362314bae61a91d4a2d04a9b679da46c082dd42R151